src/chunk_dispatch_state.c

```diff
@@ -100,12 +99,15 @@ chunk_dispatch_exec(CustomScanState *node)
 				 * (See:
 				 * https://github.com/postgres/postgres/commit/ee0a1fc84eb29c916687dc5bd26909401d3aa8cd).
 				 */
+				MemoryContextSwitchTo(estate->es_query_cxt);
```

Comment:
I don't quite understand this change : ts_chunk_dispatch_get_chunk_insert_state allocates everything i in qry_context. So   cis->arbiter_indexes should already be in the qry_context. what causes the failure here?